PLUB/Sources/Views/Meeting/MeetingViewController.swift

```diff
@@ -170,7 +170,7 @@ extension MeetingViewController: UICollectionViewDelegate, UICollectionViewDataS
   func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
     if indexPath.row < meetingList.count {
       // 플러빙 메인
-      let vc = MainPageViewController()
+      let vc = MainPageViewController(plubbingID: 0)
       vc.navigationItem.largeTitleDisplayMode = .never
```

Comment:
```suggestion
      vc.navigationItem.largeTitleDisplayMode = .never
      vc.hidesBottomBarWhenPushed = true
```
해당 코드도 추가해주시면 감사하겠습니다!

Reply:
넵 추가하여 다시 푸시하였습니다 ✔️✔️✔️